scipy/stats/_continuous_distns.py

```diff
@@ -8403,6 +8403,9 @@ class reciprocal_gen(rv_continuous):
     >>> ax.set_xticklabels(ticks)  # doctest: +SKIP
     >>> plt.show()
 
+    Details related to the decision of not defining
+    the surival function for this distribution can be
+    found in the PR: https://github.com/scipy/scipy/pull/18614
```

Comment:
Please make this a (code) comment to developers reviewing the code, not users reviewing the documentation. Users would likely take this to mean that the survival function of this distribution is not implemented, which is not what we mean to convey. They don't need to know whether `_sf` is overridden; only developers do.